docs/CONFIG.md

```diff
@@ -3993,13 +3993,13 @@ TipCapMin = '1 wei'
 
 [GasEstimator.BlockHistory]
 BatchSize = 25
-BlockHistorySize = 12
+BlockHistorySize = 1000
```

Comment:
Why increase the number of blocks used to calculate gas?

Reply:
That was an accidental change I made at some point yesterday and rolled back soon after. It's not changed in the actual defaults, must be that I forgot to regenerate the config?

Reply:
@patrickhuie19 Thanks for catch, I've re-run "make config-docs" which removed those changes.  Approve?

Reply:
Sorry just saw this, but LGTM! - my github notifications are not surfacing replies to comments :/